contract/src/main.rs

```diff
@@ -954,21 +926,44 @@ pub extern "C" fn transfer() {
     }
 
     // Update to_account owned_tokens. Revert if owned_tokens list is not found
-    match utils::get_token_identifiers_from_dictionary(&identifier_mode, &source_owner_item_key) {
-        Some(mut owned_tokens) => {
-            // Check that token_id is in owned tokens list. If so remove token_id from list
-            // If not revert.
-            if let Some(id) = owned_tokens.iter().position(|id| *id == token_identifier) {
-                owned_tokens.remove(id);
-            } else {
-                runtime::revert(NFTCoreError::InvalidTokenOwner)
-            }
-            utils::upsert_token_identifiers(&identifier_mode, &source_owner_item_key, owned_tokens)
-                .unwrap_or_revert();
-        }
-        None => runtime::revert(NFTCoreError::InvalidTokenIdentifier),
+    let token_number = utils::get_token_number(&token_identifier);
```

Comment:
change various names to align w/ page_table notation

Reply:
Changed the variable names